behaviors/paladin/holy.lua

```diff
@@ -74,67 +75,143 @@ for k, v in pairs(common.widgets) do
     table.insert(options.Widgets, v)
 end
 
+local function GetGroupUnits()
+    local group = WoWGroup(GroupType.Auto)
+    local members = {}
+
+    if group then
+        local party = group.Members
```

Comment:
`group.Members` returns all group members not only those in the current party, hence the name `party` could be misleading